SerialPrograms/Source/CommonFramework/Options/EditableTableOption.cpp

```diff
@@ -121,4 +121,10 @@ QWidget* make_double_table_cell(QWidget& parent, double& value, double min, doub
 template QWidget* make_integer_table_cell(QWidget& parent, uint16_t& value);
 template QWidget* make_integer_table_cell(QWidget& parent, int16_t& value);
 
+QWidget* make_conststring_table_cell(QWidget& parent, const QString& string) {

```

Comment:
Not really an editable but is there a way to use the table and input constant text in it ? (I could use QLabel too)